crates/re_viewport/src/space_view.rs

```diff
@@ -228,10 +251,17 @@ impl SpaceViewBlueprint {
     }
 
     #[inline]
-    pub fn set_display_name(&self, name: String, ctx: &ViewerContext<'_>) {
+    pub fn set_display_name(&self, name: Option<String>, ctx: &ViewerContext<'_>) {
```

Comment:
style nit: I think it is nice to putt context parameters early (like `&self`, which is context-like). When there are many function taking similar contexts by argument (`ViewerContext,`Ui`, etc) it is easier to ignore the first common arguments and then skip to the specific ones

Reply:
Agreed. This was done for consistency with nearby APIs. I suggest this should be added to our style guide, as we are globally inconsistent.